code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2666,96 +2666,979 @@ Follow by example and make good judgement based on length which list to include
+
+		left_hand
+			name = "Tattoo (Pawsocks, Left Hand)"
+			body_parts = list("l_hand")
+
+		right_hand
+			name = "Tattoo (Pawsocks, Right Hand)"
+			body_parts = list("r_hand")
+
+		left_leg
+			name = "Tattoo (Pawsocks, Left Leg)"
+			body_parts = list("l_leg")
+
+		right_leg
+			name = "Tattoo (Pawsocks, Right Leg)"
+			body_parts = list("r_leg")
+
+		left_foot
+			name = "Tattoo (Pawsocks, Left Foot)"
+			body_parts = list("l_foot")
+
+		right_foot
+			name = "Tattoo (Pawsocks, Right Foot)"
+			body_parts = list("r_foot")
+
+	tat_silverburgh_left
+		name = "Tattoo (Silverburgh, Left Leg)"
+		icon_state = "tat_silverburgh"
+		body_parts = list("l_leg")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_silverburgh_right
+		name = "Tattoo (Silverburgh, Right Leg)"
+		icon_state = "tat_silverburgh"
+		body_parts = list("r_leg")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_tamoko
+		name = "Tattoo (Ta Moko, Face)"
+		icon_state = "tat_tamoko"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell")
+
+	tat_tatyana_left
+		name = "Tattoo (Tatyana, Left Eye)"
+		icon_state = "tat_tatyana_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_tatyana_right
+		name = "Tattoo (Tatyana, Right Eye)"
+		icon_state = "tat_tatyana_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_tiger
+		name = "Tattoo (Tiger Stripes, All)"
+		icon_state = "tat_tiger"
+		body_parts = list("l_foot","r_foot","l_leg","r_leg","l_arm","r_arm","l_hand","r_hand","groin","chest")
+		species_allowed = list("Human","Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+		chest
+			name = "Tattoo (Tiger Stripes, Chest)"
+			body_parts = list("chest")
+
+		groin
+			name = "Tattoo (Tiger Stripes, Groin)"
+			body_parts = list("groin")
+
+		left_arm
+			name = "Tattoo (Tiger Stripes, Left Arm)"
+			body_parts = list("l_arm")
+
+		right_arm
+			name = "Tattoo (Tiger Stripes, Right Arm)"
+			body_parts = list("r_arm")
+
+		left_hand
+			name = "Tattoo (Tiger Stripes, Left Hand)"
+			body_parts = list("l_hand")
+
+		right_hand
+			name = "Tattoo (Tiger Stripes, Right Hand)"
+			body_parts = list("r_hand")
+
+		left_leg
+			name = "Tattoo (Tiger Stripes, Left Leg)"
+			body_parts = list("l_leg")
+
+		right_leg
+			name = "Tattoo (Tiger Stripes, Right Leg)"
+			body_parts = list("r_leg")
+
+		left_foot
+			name = "Tattoo (Tiger Stripes, Left Foot)"
+			body_parts = list("l_foot")
+
+		right_foot
+			name = "Tattoo (Tiger Stripes, Right Foot)"
+			body_parts = list("r_foot")
+
+	tat_tramp
+		name = "Tattoo (Tramp, Lower Back)"
+		icon_state = "tat_tramp"
+		body_parts = list("chest")
+		species_allowed = list("Human")
+
+	tat_trampheart
```

Comment:
Get rid of this dear god

Reply:
What's the kerfuffle against tramp stamps?